### DISCUSSION TITLE
Staging and release processes

### DISCUSSION THREAD

**Discussion by markcaron** [RedHat-UX/red-hat-design-system#1460] [General]
Should we create "staging" branches for our milestones/releases?

**Ideas:**
- Collect all planned release PRs into branches named after their Pokemon milestone name, like `staging/chansey`
- Collect any unplanned releases (e.g. bugs, patches, etc.) into branches named after their Pokemon evolutions, like `staging/metapod` (for Caterpie)

This should allow us to target specific releases in the PR, and avoid collisions feature sets that aren't going out yet.

Discuss...

**Comment by bennypowers** [comment]
We resolved that this is a good idea and we'll start with the upcoming release: https://github.com/RedHat-UX/red-hat-design-system/pull/1461

patches / hotfixes should target the main branch, as we've done until now

new features that we're queueing up for chansey should target the `staging/chansey` branch
